build-system/tasks/runtime-test.js

```diff
@@ -203,7 +203,13 @@ gulp.task('test', 'Runs tests', argv.nobuild ? [] : ['build'], function(done) {
     util.log(util.colors.yellow(
         'Shutting down test responses server on localhost:31862'));
     server.emit('kill');
-    done(exitCode);
+    if (exitCode && typeof exitCode !== 'Error') {
```

Comment:
Just tried it, there's nothing in the stack. Karma merely returns an error code. Nothing left to do here.